php/docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:18.04

```

Comment:
there's no php basic docker image ?

Reply:
Hmm, probably yes. The thing is: I'm not a big fan of docker, so this script was not made by me. Also has the problem that this driver version only works at PHP 7.1 and needs a bunch of steps to install.

If you have any idea of how improve it, I'll gladly implement it.

Reply:
The official docker images are debian based, so it shouldn't be that far off from what you are doing here.